README.md

```diff
@@ -12,6 +12,12 @@
 - Ruby 2.6.1
 - postgresql-9.6 postgresql-contrib-9.6
 
+#### Linux
+
+To be able to bundle on linux (for the `pg` gem) you need the postgres header files:
```

Comment:
This is already be covered in step 0 of setup  
https://github.com/DFE-Digital/manage-courses-backend/blob/master/README.md#native-1

Reply:
I'm sorry I don't see it. Could you screenshot and highlight perhaps?

Reply:
This is off of the [Go Rails](https://gorails.com/setup/ubuntu/19.04#postgresql) site that's linked-to from step 0.

<img width="834" alt="image" src="https://user-images.githubusercontent.com/15608/63955135-2d85a700-ca7c-11e9-9a00-fa4b5e81781e.png">

Reply:
on a separate note when we upping 9.6 to 11?